lib/src/view/design_main_strand.dart

```diff
@@ -971,16 +962,18 @@ Future<void> ask_for_assign_dna_sequence(
   List<DialogItem> items = [null, null, null, null, null, null, null];
 
   items[idx_sequence] =
-      DialogTextArea(label: 'sequence', value: strand.dna_sequence ?? '', rows: 10, cols: 80);
-  items[idx_use_predefined_dna_sequence] = DialogCheckbox(label: 'use predefined DNA sequence');
+      DialogTextArea(label: 'sequence', value: strand.dna_sequence ?? '', rows: 4, cols: 80);
```

Comment:
Question: what's the intention of changing the number of rows from 10 to 4?

Reply:
> Question: what's the intention of changing the number of rows from 10 to 4?

On my laptop screen, 10 rows pushes the dialog off the bottom of the page.